Packs/PowershellRemoting/Integrations/PowerShellRemoting/PowerShellRemoting.yml

```diff
@@ -0,0 +1,393 @@
+    outputs:
+      - contextPath: PsRemote.CommandResult
+        description: The stop ETL command results.
+        type: string
+      - contextPath: PsRemote.EtlFileName
+        description: The file name for the ETL
+        type: string
+      - contextPath: PsRemote.EtlFilePath
+        description: The file path and name for the ETL
+      - contextPath: PsRemote.Host
+        description: The Host the command was invoked on.
+        type: string
+      - contextPath: PsRemote.FQDN
+        description: The Fully Qualified Domain Name of the host the command was invoked on.
+        type: string
+  - arguments:
+    - description: The hostname on which to create the ETL file. For example myhost.
+        The user can provide a comma delimited list of values.
+      isArray: false
+      name: host
+    - description: The IP of the host on which to run the command. For example 1.1.1.1.
+        The user can provide a comma delimited list of values.
+      isArray: false
+      name: ip
+    - defaultValue: all
+      description: The registry key or hive to export. For example reg_key_hive=`HKEY_LOCAL_MACHINE`.
+        If the user specifies all as the value. The entire registry will be exported.
+        Default value is all.
+      name: reg_key_hive
+    - description: The path and name on the windows host where the registry file will
+        be created. For example, file_path=c:\hklm.reg.
+      name: file_path
+      required: true
+    description: This command exports the specified Registry of hive to a file.
+    name: ps-remote-export-registry
+    outputs:
+    - contextPath: PsRemote.Command.Result
+      description: The command results in the remote host - regedit for all or reg export.
+      type: string
+    - contextPath: PsRemote.Command.RegFilePath
+      description: The file path and name for the registry file.
+      type: string
+    - contextPath: PsRemote.Command.RegFileName
+      description: The file name for the registry file.
+      type: string
+    - contextPath: PsRemote.Command.Name
+      description: The name of the command that ran on the host.
+      type: string
+    - contextPath: PsRemote.Fqdn
+      description: The Full Qualified Domain Name of the host.
+      type: string
+    - contextPath: PsRemote.Host
+      description: The Host the command was invoked on.
+      type: string
+    - contextPath: PsRemote.FQDN
+      description: The Fully Qualified Domain Name of the host the command was invoked on.
+      type: string
+  - arguments:
+    - description: Specify host(s) to retrieve data from.
+      isArray: false
+      name: host
+    - description: The IP of the host on which to run the command. For example 1.1.1.1.
+        The user can provide a comma delimited list of values.
+      isArray: false
+      name: ip
+    - description: Specify a volume to retrieve its master file table.
+      name: volume
+      required: true
+    - defaultValue: "false"
```

Comment:
@DeanArbel Is the intentional?

Reply:
Nope, the volume shouldn't be required, and it shouldn't be with this default value, nice catch.